.circleci/config.yml

```diff
@@ -758,6 +758,15 @@ jobs:
           version_path: versions/cwf_operator_version
       - magma_slack_notify
 
+  ###TeraVM Run_Status & Upgrade_Run_3GPP_Tests
+  Upgrade_Run_3GPP_Tests_teravm:
```

Comment:
instead of declaring a job let's create a command like we did here: https://fb.workplace.com/groups/osssupport/permalink/5013006005414641/ and call it teravm, inside it we will create all the functions we need like upgrade_3gpp, test_if_booked etc..